Userland/Utilities/unzip.cpp

```diff
@@ -124,7 +127,25 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     }
 
     auto success = zip_file->for_each_member([&](auto zip_member) {
-        return unpack_zip_member(zip_member, quiet) ? IterationDecision::Continue : IterationDecision::Break;
+        bool keep_file = false;
+
+        if (file_filters.size() != 0) {
+            for (auto& filter : file_filters) {
+                // Convert underscore wildcards (usual unzip convention) to question marks (as used by StringUtils)
+                String string_filter = AK::StringUtils::replace(filter, "_", "?");
+                if (AK::StringUtils::matches(zip_member.name, string_filter)) {
```

Comment:
Same here:
```suggestion
                if (zip_member.name.matches(string_filter, CaseSensitivity::CaseSensitive) {
```
According to my tests with `unzip` on Linux, this should be case-sensitive.